docs/source/topics/proc_deploying-sample-application-with-odo.adoc

```diff
@@ -37,45 +37,22 @@ $ mkdir sample-app
 $ cd sample-app
 ----
 
-. Clone an example Node.js application:
+. Bootstrap a Node.js component and download a starter project:
 +
 [subs="+quotes,attributes"]
 ----
-$ git clone https://github.com/openshift/nodejs-ex
-$ cd nodejs-ex
+$ odo init --name sample-app --devfile nodejs --starter nodejs-starter
 ----
 
-. Add a `nodejs` component to the application:
+. Developing the `nodejs` application continuously:
 +
 [subs="+quotes,attributes"]
 ----
-$ odo create nodejs
+$ odo dev
 ----
-
-. Create a URL and add an entry to the local configuration file:
-+
-[subs="+quotes,attributes"]
-----
-$ odo url create --port 8080
-----
-
-. Push the changes:
-+
-[subs="+quotes,attributes"]
-----
-$ odo push
-----
-+
-Your component is now deployed to the cluster with an accessible URL.
-
-. List the URLs and check the desired URL for the component:
 +
-[subs="+quotes,attributes"]
-----
-$ odo url list
-----
-
-. View the deployed application using the generated URL.
+You can now access the application in your local browser using the URL from the output of the above command and start your development loop.
+[command]`odo` will watch for changes and push the code for real-time updates.
```

Comment:
imo "developing continuously", "start your development loop" is a bit mysterious/begs for more details. Maybe we should just link to https://odo.dev/docs/user-guides/quickstart/nodejs and only detail the "Login to OpenShift Cluster" step as it's a bit more crc specific?